tests/query/dnss/test_nameserver.py

```diff
@@ -485,14 +485,14 @@ def fake_get_default_resolver(*args, **kwargs):
         get_default_resolver_patch.side_effect = fake_get_default_resolver
 
         expected_nameserver = [
-            "1.1.1.1",
-            "8.8.8.8",
-            "9.9.9.9",
+            "9.9.9.10",
+            "149.112.112.10",
+            "2620:fe::10, 2620:fe::fe:10"
```

Comment:
Tests are failing because the IPv6 is wrong + not listed into PyFunceble/quey/dns/nameserver.py.